### PR TITLE
Fix Eigen3

### PR DESCRIPTION
**Pull request by davetcoleman**
Add C++11 support to fix compile bug
Remove unneeded and missing dependency on ompl_visual_tools

### CODE REVIEW

**Comment by v4hn** [comment]
building fails. The ikfast plugin is missing an explicit namespace to disambiguate `isnan`

File: baxter/baxter_ikfast_left_arm_plugin/CMakeLists.txt [hunk 1]
```diff
@@ -24,7 +26,14 @@ catkin_package(
   tf_conversions
 )
 
-include_directories(include)
+include_directories(SYSTEM
+  ${EIGEN3_INCLUDE_DIRS}
+)
```
Comment (v4hn) [inline baxter/baxter_ikfast_left_arm_plugin/CMakeLists.txt]: maybe remove the `SYSTEM` given https://github.com/ros-planning/geometric_shapes/pull/55?

File: baxter/baxter_ikfast_right_arm_plugin/CMakeLists.txt [hunk 2]
```diff
@@ -24,7 +26,14 @@ catkin_package(
   tf_conversions
 )
 
-include_directories(include)
+include_directories(SYSTEM
+  ${EIGEN3_INCLUDE_DIRS}
+)
```
Comment (v4hn) [inline baxter/baxter_ikfast_right_arm_plugin/CMakeLists.txt]: same here

Comment (davetcoleman) [reply baxter/baxter_ikfast_right_arm_plugin/CMakeLists.txt]: done

**Comment by davetcoleman** [comment]
ready to be merged

**Pull request merged**